Source/EPUBCore/FREpubParser.swift

```diff
@@ -106,7 +107,7 @@ class FREpubParser: NSObject {
             try readContainer()
             try readOpf()
         } catch {
-            print(error.localizedDescription)
+            os_log("readEpub error: %{public}@", error.localizedDescription)
```

Comment:
I wonder if this `{public}` access modifier should be applied to all of the logs with variables printed out. Or if they should be `{private}`. 🤷🏻‍♂️

Reply:
will apply to all